pjsip-apps/src/pjsua/pjsua_app.c

```diff
@@ -666,6 +666,54 @@ static void on_buddy_state(pjsua_buddy_id buddy_id)
 }
 
 
+/*
+ * Handler on buddy blf state changed.
```

Comment:
When seeing this sentence, the first question is: what is blf? Better not using abbreviations.
Even better: not using abbreviations in the function name in the first place.

Reply:
Sam for the other functions.

Reply:
Sure. I changed the abbreviation from the functions description by full feature name.